gap/DocumentationTree.gi

```diff
@@ -442,12 +482,46 @@ InstallMethod( SectionInTree, [ IsTreeForDocumentation, IsString, IsString ],
     return StructurePartInTree( tree, [ chapter_name, section_name ] );
 end );
 
+## This method creates the section and if it is not already in the tree,
```

Comment:
"... creates the section ..." what is "the" section this is referring to? Perhaps "a" section is meant?

Also, it doesn't always create a section, does it? So perhaps:  "This method returns a section object for [INSERT MEANINGFUL TEXT, like "... for the given <context>" -- but note that this example may be nonsense, I am not anymore familiar with the code here....]. If the section already exists, this is returned; otherwise, a new section node is created and returned. In the latter case, it is added as a child..."